code/game/objects/items/weapons/fishing_rod.dm

```diff
@@ -0,0 +1,130 @@
+#define TIME_TO_CATCH = 80
+
+// -----------------------------
+//         Bait Datums
+// -----------------------------
+
+// Base datum for bait_type
+// bait: The object to be used as bait
+// fish: list of fish you can catch and their probability
+//       uses the format list(list(<fish>, <absolute probability>), ...)
```

Comment:
Doing `list(<fish> = <absolute_probability>)` might be a better idea, unless you plan on having more data stored here.